core/src/validator/__tests__/test.case.ts

```diff
@@ -175,33 +164,36 @@ Deno.test("KeyLowerCase - withWarning", async () => {
   }
 
   for (const asset_metadata of metadata) {
-    await mainValidator.Execute(
-      asset_metadata.asset_name,
-      asset_metadata,
-      metadata,
-    );
+    mainValidator.Execute(asset_metadata.assetName, asset_metadata, metadata);
   }
 
   const result = mainValidator.GetResults();
 
-  console.debug(JSON.stringify(result));
+  // console.debug(JSON.stringify(result));
```

Comment:
remove commented clog

Reply:
Done, when it will be merged, I will also add a global logger to remove the remaining Clogs